deploy/kubernetes/base/server-service.yaml

```diff
@@ -9,11 +9,13 @@ spec:
   - name: "grpc-api"
     port: 443
     targetPort: 8080
+    nodePort: 30080
   - name: "http-metrics"
     port: 8081
     targetPort: 8081
+    nodePort: 30081
   selector:
     io.kompose.service: server
-  type: NodePort
+  type: NodePort # Required for ingress
```

Comment:
why was it already there, then?

Reply:
It was already there in the base config because that's how kubernetes in docker connected to it.
It was previously overridden to be a `LoadBallancer` in GCE. I thought it would be good to document what depends on it being a `NodePort`

Reply:
Thanks for the clarification - it's a good idea to explain why this type is required.

Now all flavours, including baremetal are `type: NodePort`, as I can't see it being overridden (well, kustomised). Correct?